.github/workflows/CI.yml

```diff
@@ -12,15 +15,26 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          # Test on the LTS, Invenia prod release and latest release
```

Comment:
These should stay because we're only testing on linux and maybe x86 in the future.